tests/performance-test/performance-test.sh

```diff
@@ -277,25 +266,25 @@ while true; do
         ;;
         "RESULTS")
             printf "\n*** Collecting test results ***\n"
-            PODNAME=$(oc get pod -l job-name=saf-perftest-notify -o jsonpath='{.items[0].metadata.name}')
-            oc exec "$PODNAME" -- pkill collectd > /dev/null
-            sleep 3
-            oc cp "sa-telemetry/${PODNAME}:/tmp/events.json" /tmp/events.json
+            # PODNAME=$(oc get pod -l job-name=saf-perftest-notify -o jsonpath='{.items[0].metadata.name}')
```

Comment:
Are you disabling events for now because SAF2 doesn't run ElasticSearch yet?

Reply:
Yes. As soon as it's working I'll re-test it all and uncomment. I do feel a bit guilty leaving commented code in the file, but it should be very temporary.